packages/twenty-front/src/modules/ui/layout/selectable-list/hooks/useSelectableList.ts

```diff
@@ -30,11 +31,32 @@ export const useSelectableList = (props?: UseSelectableListProps) => {
   );
   const isSelectedItemId = useRecoilValue(isSelectedItemIdSelector);
 
+  const resetSelectedItemIdState = useResetRecoilState(selectedItemIdState);
+  const resetSelectableItemIdState = useResetRecoilState(
+    selectableItemIdsState,
+  );
+  const resetSelectableListOnEnterState = useResetRecoilState(
+    selectableListOnEnterState,
+  );
+  const resetIsSelectedItemIdSelector = useResetRecoilState(
+    isSelectedItemIdSelector,
+  );
+
+  const resetSelectedItem = () => {
+    resetSelectedItemIdState();
+    resetIsSelectedItemIdSelector();
+  };
+
   return {
     setSelectableItemIds,
     isSelectedItemId,
     setSelectableListOnEnter,
     selectableListId: scopeId,
     isSelectedItemIdSelector,
+    resetSelectedItemIdState,
```

Comment:
we should not expose these as we don't want it to be part of the external API of useSelectableList, 
we should only expose resetSelectedItem